cmr/queries.py

```diff
@@ -339,18 +391,18 @@ def online_only(self, online_only: bool = True) -> Self:
         return self
 
     def _format_date(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike]
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike]
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike]
```